schemas/json/theme.json

```diff
@@ -1023,15 +1023,35 @@
 					"properties": {
 						"background": {
 							"description": "Sets the `background-color` CSS property.",
-							"type": "string"
+							"oneOf": [
+								{ "type": "string" },
+								{ "type": "object" }
+							],
+							"properties": {
```

Comment:
Does this `properties` definition need to exist on the object of `type: object` in the `oneOf` array above?

Reply:
Ah yes, I think so, based on some other examples. Thanks!